sdk/storage/azure-storage-blobs/test/ut/blob_sas_test.cpp

```diff
@@ -10,6 +10,37 @@
 
 namespace Azure { namespace Storage { namespace Test {
 
+  static std::string AppendQP(Azure::Core::Url url, std::string qp)
+  {
+    std::string absoluteUrl = url.GetAbsoluteUrl();
+    auto existingQP = url.GetQueryParameters();
+    bool startWithQuestion = qp[0] == '?';
```

Comment:
What if `qp` is empty?

Reply:
This is in tests. Why would it be empty? If it's empty, don't call the method :)